oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,428 @@
+communication, firmness and directness are equally important.
+Maintainers must enforce the project's standards.  In cases where a
+proposed change isn't aligned with the our standards or direction, it
+is best to make that clear as early as possible.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Selection and approval of Core Contributors with commit rights to their repositories
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards, both technical and social: maintainers
+  must remain friendly, kind, and open, but that should not supercede
+  enforcement of technical standards
+* The ability to solve or escalate issues facing users of the
+  component
+
+Technical responsibilities of maintainers include:
+
+* Understanding the current component architecture and implementation
+* Ensuring that basic documenation for the component exists and is
+  accurate
+* Having a documented technical vision for the evolution of the
+  component
+* The ability to assess the appropriateness of proposed changes to or
+  extensions of the component
+* The role of the component in the overall platform and how it
+  interacts with other components
+* Must be available to participate in the comment period for Core Contributor
+  nominees nominated to commit code to the maintainer's repo(s), as part of the
+  `Core Contributor Program`_
+  
+It is acceptable for a component to be maintained by a team or group
+as long as that arrangement does not reduce the reachability of at least one
+maintainer or reduce accountability for the above responsibilites. The
+maintaining team must have ways of direct contact, such as an email address and
+a GitHub team, that anyone can use to get ahold of someone - it is completely
+acceptable for the team to figure out arrangements for list monitoring, such as
+an "on-call" rotation.
+
+Prerequisites
+-------------
+In order to be a repository maintainer, you must
+
+* Be an active member of the Open edX community who has made technical
+  contributions to the project in the last 6 months.  The types of
+  contributions that are relevant will match those defined in the
+  `code contributors`_
+* Abide by the Open edX `Code of Conduct`_
+  <https://openedx.org/code-of-conduct/>`_
+* Be a member of the `Core Contributor Program`_ with rights to the repository
+  in question who has demonstrated excellence in the role of core contributor.
+  From time-to-time, in the case of a strong candidate and project need, core
+  contributor status and maintainer status may be conferred at the same time.
+* In the case of repos being moved into the `openedx GitHub organization`_, be
+  an original maintainer of the code.
+
+Bootstrapping
+-------------
+In order to ensure an orderly roll-out of the formal maintainer role,
```

Comment:
I'm writing this comment just to explicitly say out loud what I believe our shared expectations already are.  I imagine they may not be obvious to folks who haven't been intimately involved in the edX -> tCRIL transition process.

The sentence here about roll-out for existing 2U staff makes sense to me, under the assumption that - as @sarina says below - we expect to codify the exact timelines and compliance expectations elsewhere in a more nuanced way.  I don't think it's anyone's expectation that the moment we switch from "owners" to "maintainers" we're going to start a clock and start removing maintainers one month later for not bringing their repositories up to compliance with all our standards.  

Rather, the transition has always had an expectation that we will "do no harm" to the edX business as we transition edX from the "owner" to a true peer maintainer.  In fact, I'd expect this goes for other businesses too as they onboard to maintainership.

I would hope that transitioning 2U staff from owners to maintainers is unconcerning and almost a no-op in the short term for those of us on the 2U side, but that we apply pressure in the medium-term to ensure we prioritize bringing the platform - regardless of maintainer - up to our standards.  Once we have a reasonable expectation of quality in most quarters, then I'd expect the "one month" clock to start coming into play.

Again, my assumption is that none of that is controversial, but isn't called out explicitly here and probably doesn't need to be.

Reply:
Thanks @davidjoy I agree with your framing.  We have flexibility in terms of when we expect things to be done and the idea is not to create an overwhelming body of new work.  The transition needs to be orderly and I do believe that much of the needed work is being done today.  Will there be some new work?  Well yes, there will be a little, but I think it will benefit all users of the project and set the stage of distributed, shared maintenance.

Reply:
Agree @davidjoy . I want to double-check that there's no action needed on this comment?

Reply:
👍🏻 Yeah!  No action needed.